site/content/installation/ngf-images/building-the-images.md

```diff
@@ -57,14 +57,14 @@ If building the NGINX Plus image, you will also need a valid NGINX Plus license
      make PREFIX=myregistry.example.com/nginx-gateway-fabric build-nginx-image
      ```
 
-   - To build just the NGINX Plus image:
+   - To build just the NGINX Plus image, replace `<my-docker-registry>` with your registry's path:
```

Comment:
All of the previous examples don't have this disclaimer, so I don't feel like we need it in this case either. The example registry is pretty clear in my opinion.

Reply:
updating